packages/profiling-node/bindings/cpu_profiler.cc

```diff
@@ -111,14 +111,13 @@ class MeasurementsTicker {
 
   size_t listener_count();
 
-  ~MeasurementsTicker() {
+  void Cleanup() {
```

Comment:
I think you should also make sure checking for ref using `uv_has_ref` and you need to call unref here.

Reply:
On line 92, you call `uv_unref` but it should be `uv_ref` to keep the event loop alive.